source/gui/_addonStoreGui/viewModels/store.py

```diff
@@ -241,6 +241,21 @@ def _makeActionsList(self):
 				validCheck=lambda aVM: isinstance(aVM.model, _AddonStoreModel),
 				actionTarget=selectedListItem
 			),
+			AddonActionVM(
+				# Translators: Label for an action that opens the webpage to see and send feedback for the selected add-on
+				displayName=pgettext("addonStore", "See or provide &feedback"),

```

Comment:
My same comment here as in the documentation: in my opinion this should be "Read", not "See". Or, the entire option should just be called "Reviews"--this menu tends to use short options.

Reply:
I agree - this should just be "Reviews"

Reply:
@seanbudd and @XLTechie, "Review" is not suitable IMO:
* "Review" may let think to a developer's review. This is more true for add-ons given reviewing add-ons was part of the process to publish add-ons on the old website. This can lead to confusion.
* At least in French, I would have difficulty to translate "Review" so that people understand that it is for anybody.

I would prefer the longer but more explicit "Read and provide feedback". But if you stick to something shorter, "Feedback" seems more suitable than "Review".

Of course, an alternative is also to merge the PR as is and wait for feedback from alpha users and translators. The UG and the GUI can be changed later if a more appropriate term is found.

Reply:
I agree with Cyrille: reviews sounds like the technical procedure for the old website, and feedback is mor understandable, imo, at least in certain languages; anyway imo this shouldn"t prervent to merge this pr.